roast.js

```diff
@@ -13,18 +19,38 @@ Roast.prototype.it = function it(description, testFunction) {
 };
 
 Roast.prototype.run = function run() {
-    var testedCount = 0;
     this.tests.forEach(function runTest(test) {
+        if (this.teamcityReporter.isEnabled()) {
```

Comment:
I wonder if we could have a reporter and just check once for whether we assign the teamcity reporter to it, or the regular console reporter. Then we could avoid all the `&& this.teamcityReporter.IsEnabled()` checks whenever we log/report a result. It would also make it easier to add other reporters.